src/utils/php_binaries.rs

```diff
@@ -7,6 +7,30 @@ use std::process::Command;
 use std::process::Stdio;
 use std::str;
 
+pub(crate) fn current() -> String {
+    let binaries = all();
+
+    let binaries_regex = if cfg!(target_family = "windows") {
+        // On Windows, we mostly have "php" and "php-cgi"
+        Regex::new(r"php(\d+(\.\d+))?(-cgi)\.exe$").unwrap()
+    } else {
+        // This will probably need to be updated for other platforms.
+        // This matches "php", "php7.4", "php-fpm", "php7.4-fpm" and "php-fpm7.4"
+        Regex::new(r"php(\d+(\.\d+))?(-fpm)(\d+(\.\d+))?$").unwrap()
```

Comment:
`-fpm` shouldn't be required, should it?
https://regexr.com/57ffn
```suggestion
        Regex::new(r"php(\d+\.\d+)?(-fpm)?(\d+\.\d+)?$").unwrap()
```

Reply:
Well for now I don't have a way to specifically categorize PHP versions like the Symfony CLI does.

For now, I want to focus on `php-fpm` on non-windows, and fallback to the native `php` binary.

To be more efficient, the `binaries_from_path` function must return something else than a list of strings, and instead I should create some kind of `PhpVersion` struct that will hold a list of information similar to the one Symfony returns:

```
 symfony local:php:list
┌─────────┬──────────────┬─────────┬─────────┬─────────────┬─────────┬─────────┐
│ Version │  Directory   │ PHP CLI │ PHP FPM │   PHP CGI   │ Server  │ System? │
├─────────┼──────────────┼─────────┼─────────┼─────────────┼─────────┼─────────┤
│ 7.4.2   │ E:\dev\php74 │ php.exe │         │ php-cgi.exe │ PHP CGI │ *       │
└─────────┴──────────────┴─────────┴─────────┴─────────────┴─────────┴─────────┘
```

As this project is now in very (very) early stage, I first want to focus on php-cgi on windows and php-fpm on others in order to have a working PoC, and after that move to proper versions :) This regex will be remove that day 😉